apps/examples/testcase/le_tc/kernel/tc_semaphore.c

```diff
@@ -19,20 +19,24 @@
 /// @file semaphore.c
 /// @brief Test Case Example for semaphore API
 #include <tinyara/config.h>
+#include <tinyara/clock.h>
 #include <stdio.h>
 #include <errno.h>
 #include <sys/time.h>
 #include <sys/types.h>
 #include "tc_internal.h"
 
-#define PSHARED     0
-#define SEC_2       2
-#define LOOP_CNT    5
+#define PSHARED			0
+#define SEC_2			2
+#define LOOP_CNT		5
+#define SEM_PRIO_DEFAULT	3
 
 static sem_t g_empty;
 static sem_t g_full;
 static sem_t g_sem;
 
+int sem_tickwait(FAR sem_t *sem, systime_t start, uint32_t delay);
```

Comment:
why did you declare here?
this prototype is in tinyara/semaphore.h

Reply:
Yesterday, that was added. Could you use latest version and include that header instead of this as Jeongchan said?